src/components/Header/Header.scss

```diff
@@ -1,9 +1,142 @@
 @import '~style/variables';
 
-.wrapper {
-  border: 1px solid blue;
+header {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-primary--500;
+  color: $color-white;
+
+  a {
+    color: $color-white;
+    text-decoration: underline;
+    text-decoration-color: $color-highlight;
+  }
+}
+
+.govContainer {
+  display: flex;
+  height: 48px;
+  padding-left: 14px;
+  flex-flow: column wrap;
+  align-items: flex-start;
+  align-content: flex-start;
+  justify-content: center;
+
+  @include for-desktop-up {
+    padding-left: 30px;
+    height: 32px;
+    align-items: center;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    background-color: $color-primary--400;
+
+    > * {
+      margin-right: 8px;
+    }
+  }
+}
+
+.compactTitle,
+.compactSubtitle {
+  @include for-desktop-up {
+    display: none;
+  }
+}
+
+.govAu,
+.govWebsite {
+  display: none;
+
+  @include for-desktop-up {
+    display: initial;
+  }
+}
+
+.govAu {
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.govWebsite {
+  font-size: 12px;
+  font-weight: 100;
+}
+
+.mainContainer {
+  display: none;
+
+  @include for-desktop-up {
+    padding: 0 32px;
+    max-height: 176px;
+    display: flex;
+    flex-flow: column wrap;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+}
+
+.star {
+  height: 22px;
+  width: 22px;
+  max-width: 22px;
+  flex: 0 0 100%;
+  margin-right: 5px;
+
+  @include for-desktop-up {
+    height: 18px;
+    width: 18px;
+    flex: 0 1 auto;
+  }
+}
+
+.projectStatus {
+  margin-top: 24px;
+  size: 16px;
+  text-align: center;
+  width: 80px;
+  font-weight: 600;
+  border: 1px solid $color-badge;
+  border-radius: 17px;
+  background-color: $color-badge;
 }
 
 .title {
-  font-size: 20px;
+  font-size: 36px;
+  font-weight: 300;
+  line-height: 46px;
+  margin-top: 10px;
+  max-height: initial;
+}
+
+.compactTitle {
+  font-size: 14px;
+  font-weight: 600;
+  flex: 0 0 16px;
+}
+
+.subtitle {
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+.compactSubtitle {
+  text-transform: uppercase;
+  font-weight: 200;
+  font-size: 10px;
+  flex: 0 0 14px;
+}
+
+.orgLink {
+  margin-top: 14px;
+  align-self: flex-end;
+}
+
+.orgLogo {
+  height: 65px;
+  width: 102px;
+}
+
+.contactLink {
+  font-size: 22px;
+  align-self: flex-end;
```

Comment:
My bad for not testing this in IE till now, I really should have done it all in one go... but it's broke:

![image](https://user-images.githubusercontent.com/4443482/31210052-d4869948-a9da-11e7-8738-d16c356ec5cf.png)

There's a few things:
* I told you about `initial`, not realising it's [not supported in IE](https://developer.mozilla.org/en-US/docs/Web/CSS/initial). Fixing this (I'll put other comments inline) fixes the drop shadows showing and the top text not showing.
* The header being too tall is likely a flexbox bug. A clever chap has created a page that chronicles every bug with flexbox, worth having a read through: https://github.com/philipwalton/flexbugs.

Do you have a Windows VM? Do you know about modern.ie?